docs/quickstart/1-install-requirements.md

```diff
@@ -15,6 +15,18 @@ Follow the instructions based on your OS:
 
 ## Docker Compose
 
-**source{d} CE** is deployed as Docker containers, using [Docker Compose](https://docs.docker.com/compose), but it is not required to have a local installation of Docker Compose; if it is not found it will be deployed inside a container.
+**source{d} CE** is deployed as Docker containers, using [Docker Compose](https://docs.docker.com/compose), but it is not required to have a local installation of Docker Compose; if it is not found it will be downloaded from docker sources, and deployed inside a container.
 
-If you prefer a local installation of Docker Compose, you can follow the [Docker Compose install guide](https://docs.docker.com/compose/install)
+If you prefer a local installation of Docker Compose, or you have no access to internet to download it, you can follow the [Docker Compose install guide](https://docs.docker.com/compose/install)
```

Comment:
At first read, this sentence sounds weird as it says to follow a link if I have no access to the internet.

Reply:
It only points to the official docker-compose installation guide. It's up to the visitor to follow that guide, and how.
(the docs don't imply to be read offline, and they do not assume that `sourced` is run the same way as the docs are read)